docs/MatrixOne/Develop/import-data/bulk-load/0.7-load-jsonline.md

```diff
@@ -0,0 +1,202 @@
+
+A valid object JSONLines example:
+
+```
+{"id":1,"father":"Mark","mother":"Charlotte"}
+{"id":2,"father":"John","mother":"Ann"}
+{"id":3,"father":"Bob","mother":"Monika"}
+```
+
+Invalid object JSONLines example (with nested structure):
+
+```
+{"id":1,"father":"Mark","mother":"Charlotte","children":["Tom"]}
+{"id":2,"father":"John","mother":"Ann","children":["Jessika","Antony","Jack"]}
+{"id":3,"father":"Bob","mother":"Monika","children":["Jerry","Karol"]}
+```
+
+A valid array JSONLines example, it needs to look like a CSV format.
+
+```
+["Name", "Session", "Score", "Completed"]
+["Gilbert", "2013", 24, true]
+["Alexa", "2013", 29, true]
+["May", "2012B", 14, false]
+["Deloise", "2012A", 19, true]
+```
+
+Invalid array JSONLines example (Data type and column numbers don't match):
+["Gilbert", "2013", 24, true, 100]
+["Alexa", "2013", "twenty nine", true]
+["May", "2012B", 14, "no"]
+["Deloise", "2012A", 19, true, 40]
+
+## Syntax
+
+- Scenario 1: The data file is in the same machine with the MatrixOne server.
+
+```
+LOAD DATA INFILE
+    {'filepath'='FILEPATH', 'compression'='COMPRESSION_FORMAT', 'format'='FILE_FORMAT', 'jsondata'='object'/'array'} INTO TABLE table_name [IGNORE x LINES/ROWS];
+```
+
+- Scenario 2: The data file is in separate machines with the MatrixOne server.
+
+```
+LOAD DATA LOCAL INFILE
+    {'filepath'='FILEPATH', 'compression'='COMPRESSION_FORMAT', 'format'='FILE_FORMAT', 'jsondata'='object'/'array'} INTO TABLE table_name [IGNORE x LINES/ROWS];
+```
+
+**Parameter Description**
+
+|Parameter|Value|Required/Optional | Description|
+|:-:|:-:|:-:|:-:|
+|filepath|String| Required| The file path.|
+|compression|auto/none/bz2/gzip/lz4|Optional | Compression algorithm format. |
+|format|csv/jsonline|Optional |the loading file format. default is csv.|
+|jsondata|object/array|Optional| jsonline format. If `format` is *jsonline*, must specify *jsondata*.|
+|table_name|String|Required|table name to load into|
+|x|Number|Optional|lines to be ignored while loading|
+
+**DDL guidelines for JSONLines format data**
+
+Before load JSONLines data into MatrixOne, we need to firstly create a table. As JSON data type is not the same as MatrixOne data type, we need a guideline for DDL.
+
+|JSON Type|MatrixOne Type|
+|:-:|:-:|
+|String| VARCHAR (with a certain length limit)|
+|String| TEXT (without knowing the limit of this string)|
+|String| DATETIME or TIMESTAMP (with format as "YYYY-MM-DD HH:MM:SS.XXXXXX")|
+|String| DATE (with format as "YYYY-MM-DD")|
+|String| TIME (with format as "HH-MM-SS.XXXXXX")|
+|Number| INT (with interger numbers)|
+|Number| FLOAT or DOUBLE (with floating numbers) |
+|Boolean| BOOL(true/false)|
+|Object| Not supported|
+|Array| Not supported|
+|Null| Not supported|
```

Comment:
Object, Array & Null are all supported, `Object` and `Array` ->  must be json type, Null is ok for all type.